proposals/Level-1.md

```diff
@@ -203,80 +176,93 @@ corresponding exception.
-flushed, the embeddder defines how to handle uncaught exceptions.  Otherwise, the
+flushed, the embedder defines how to handle uncaught exceptions.  Otherwise, the
 found enclosing try block is the catching try block.
 
 A throw inside the body of a catch block is never caught by the corresponding
 try block of the catch block, since instructions in the body of the catch block
 are not in the body of the try block.
 
+As expected, it is left to the embedder to define how WebAssembly exceptions are
+handled within code defined by the embedder.
+
 Once a catching try block is found for the thrown exception, the operand stack
 is popped back to the size the operand stack had when the try block was entered,
-and then an except_ref (for the exception) is pushed back onto the stack.
+and then an except_ref referring to the caught exception is pushed back onto the
+operand stack.
 
 If control is transferred to the body of a catch block, and the last instruction
 in the body is executed, control then exits the try block.
 
 If the selected catch block does not throw an exception, it must yield the
 value(s) expected by the corresponding catching try block. This includes popping
-the caught except_ref.
+the caught exception.
 
-Note that a caught exception except_ref value can be rethrown using the
-`rethrow` instruction.
+Note that a caught exception can be rethrown using the `rethrow` instruction.
 
 ### Rethrowing an exception
 
 The `rethrow` instruction takes the exception associated with the `except_ref`
-on top of the stack, and rethrows that exception. A rethrow has the same effect
+on top of the stack, and rethrows the exception. A rethrow has the same effect
 as a throw, other than an exception is not created. Rather, the referenced
 exception on top of the stack is popped and then thrown.
 
 ### Exception data extraction
 
-The `if_except block` defines a conditional query of the exception in the
-except_ref on top of the stack. The except_ref is popped when queried. The
-if_except block has two subblocks, the `then` and `else` subblocks, like that of
-an `if` block. The then block is a sequence of instructions following the
-`if_except` instruction. The else block begins with the `else` instruction.  The
-scope of the if_except block is from the `if_except` instruction to the
-corresponding `end` instruction.
+The `if_except` block begins with an `if_except` instruction, and
+has two instruction blocks, defined by the `then` and `else` instructions
+like that of an `if` block.
+
+The `if_except` instruction queries of the exception on top of the stack, and
+then pops the exception. If the exception has the same exception tag as defined
+by the `if_except` instruction, the `then` block is entered. Otherwise the
+`else` block is entered. If the if_except block doesn't have an else block,
+control passes to the instruction following that block.
+The exception value is not popped when queried.
```

Comment:
In line 229 it says it pops the exception, but in line 233 it says the exception is not popped. In #44 I thought we decided to always pop the exception.